Core/Inc/main.h

```diff
@@ -29,7 +29,8 @@ extern "C" {
 
 /* Includes ------------------------------------------------------------------*/
 #include "stm32f4xx_hal.h"
-
+#include "stm32f4xx_hal_can.h"
```

Comment:
Same over here. Did you have to include this here? I don't see any changes to main.c so I'm not sure if you used any functions from these 2 files.